superset-frontend/src/components/Menu/LanguagePicker.tsx

```diff
@@ -17,15 +17,17 @@
  * under the License.
  */
 import React from 'react';
-import PropTypes from 'prop-types';
 import { NavDropdown, MenuItem } from 'react-bootstrap';
 
-const propTypes = {
-  locale: PropTypes.string.isRequired,
-  languages: PropTypes.object.isRequired,
-};
+export interface LanguagePickerInterface {
```

Comment:
can we name this `LangaugePickerProps`, since it's the type for the props of the react component `LanguagePicker`?

Reply:
thanks for catching this. called it Interface by mistake. renamed to Props